test/SendRecv_SinglePairs.cpp

```diff
@@ -0,0 +1,86 @@
+/*************************************************************************
+ * Copyright (c) 2022 Advanced Micro Devices, Inc. All rights reserved.
+ *
+ * See LICENSE.txt for license information
+ ************************************************************************/
+#include "TestBed.hpp"
+
+namespace RcclUnitTesting
+{
+  TEST(SendRecv, SinglePairs)
+  {
+    TestBed testBed;
+
+    // Configuration
+    std::vector<ncclDataType_t> const& dataTypes       = {ncclInt32, ncclFloat64};
+    std::vector<int>            const  numElements     = {1048576, 53327, 1024};
+    bool                        const  inPlace         = false;
+    bool                        const  useManagedMem   = false;
+    // SendRecv_AllPairs true or false?
+    int numCollPerGroup = 0;
+    bool isCorrect = true;
+    int totalRanks = testBed.ev.maxGpus;
+    for (int isMultiProcess = 0; isMultiProcess <= 1 && isCorrect; ++isMultiProcess)
+    {
+      int const numProcesses = isMultiProcess ? totalRanks : 1;
+      testBed.InitComms(TestBed::GetDeviceIdsList(numProcesses, totalRanks), 0);

```

Comment:
I don't know what happens if you set numCollectives in a group to 0.  In this case, you want the default, which is just 1.  Can just leave it blank